tools/cli/templates/projects/vite/index.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable no-unused-vars */
+import { Checkbox } from 'coherent-gameface-checkbox';
+
+import './node_modules/coherent-gameface-checkbox/coherent-gameface-components-theme.css';

```

Comment:
Can you try without the `./node_modules` since in theory it should resolve it automatically, but we are not sure :D